app/models/manageiq/providers/container_manager.rb

```diff
@@ -11,7 +11,8 @@ class ContainerManager < BaseManager
     has_many :container_image_registries, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_images, :foreign_key => :ems_id, :dependent => :destroy
     has_many :persistent_volumes, :foreign_key => :parent_id, :dependent => :destroy
-
+    has_many :container_component_statuses, :foreign_key => :ems_id, :dependent => :destroy
+    
```

Comment:
@dkorn I suspect that this extra line is adding whitespaces, can you check and remove them? Thanks.